index.html

```diff
@@ -11,15 +11,9 @@
   <body>
     
 
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
```

Comment:
This is the only thing I want included in my pull request for HTML. I forgot to link JQuery on the original HTML frame I created.